src/test/java/org/openstreetmap/atlas/tags/filters/ConfiguredTaggableFilterTest.java

```diff
@@ -58,7 +58,7 @@ public void testDefaultOsmosisRelationConfiguration()
     {
         final Resource relationResource = new InputStreamResource(
```

Comment:
Do you want rename the resource to match too?

Reply:
oh, good catch

Reply:
on closer examination, just going to remove this test entirely. the default filter no longer filters anything out, so I don't think there's a real need to test it, especially since filtering behavior is already being tested by the way filter configuration test below.